.github/repo_meta.yaml

```diff
@@ -0,0 +1,25 @@
+# point_of_contact: the owner of this repository, can be a GitHub user or GitHub team
+point_of_contact: @sfc-gh-kmcgrady
```

Comment:
Note that we may want to mark an appropriate GitHub team as the point of contact here, but I'm not sure how teams within the Streamlit organization are now set up now that our personal GitHub accounts are outside contributors.